### PR TITLE
Topic recognition - OASIS-DCGAN for COMP3710 Assignment

### PR DESCRIPTION
**Pull request by reecejocumsen**
I (Reece Jocumsen, 44786803) implemented a DCGAN based on OASIS brain data set. For usage instructions and examples, please see the README file.

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Sufficiently detailed commit messages
- Code valid for Hard Difficulty
- Results are good

Other Feedback
- Formatting bug in README. HTML tags are shown.

**Comment by rexsw** [comment]
Code looks good for hard difficulty

I have no additional feedback good job.

File: recognition/OASIS-DCGAN/dcgan.py [hunk 1]
```diff
@@ -0,0 +1,229 @@
+import os
```
Comment (shakes76) [inline recognition/OASIS-DCGAN/dcgan.py:1]: Code header block should be added for all scripts/source files

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.

**Pull request merged**